src/test/unit_tests/test_bucketspace_fs.js

```diff
@@ -68,6 +69,7 @@ const account_user2 = {
     name: 'user2',
```

Comment:
can you please add a test that checks a scenario where an account is not authorized to create a bucket?

Reply:
sure, will add that.